include/experimental/__p2630_bits/submdspan_extents.hpp

```diff
@@ -68,6 +69,12 @@ struct index_pair_like<std::tuple<IdxT1, IdxT2>, IndexType> {
                                 std::is_convertible_v<IdxT2, IndexType>;
 };
 
+template <class IdxT1, class IdxT2, class IndexType>
+struct index_pair_like<tuple<IdxT1, IdxT2>, IndexType> {
```

Comment:
Might be worth a comment "our device-compatible poor man's implementation of tuple" for the future reader that might be surprised to see that specialization interleaved with all the std:: ones.